system/lang/pl.json

```diff
@@ -99,8 +150,26 @@
       "Legacy": "Ruchy Dziedzictwa",
       "Quest": "Ruchy Zadań",
       "Relationship": "Ruchy Więzi",
-      "Suffer": "Ruchy Konsekwencji"
+      "Suffer": "Ruchy Konsekwencji",
+      "Connection": "Ruchy Komunikacji",
+      "Exploration": "Ruchy Eksploracji",
+      "Failure": "Ruchy Porażki",
+      "Rarity": "Ruchy Rzadkości",
+      "Recover": "Rychy Wypoczynku",
+      "Scene Challenge": "Scene challenge moves",
+      "Session": "Ruchy Sesji",
+      "Threat": "Ruchy Zagrożeń",
+      "Threshold": "Ruchy Progów"
+      
     },
+    "MoveContents": {
+    "Face Danger": "Zmierz się z niebezpieczeństwem"
+    },
+    "Failure": "Porażka",
+    "Threat": "Zagrożenie",
+    "Rarity": "Rzadkośc",
+    "Optional Delve": "Opcjonalna Delve",
```

Comment:
would it make sense to translate 'Delve', given that the term is translated elsewhere in this file?

Reply:
Probably i forgot, but with word "delve" I have problem. Because of context how Ironsworn using it. To be honest i didn't find any good polish word for it. Probably we will debate about it at discord #ironsworn-polish :P

Reply:
Hmm. Looking at the code, "Optional Delve" may not actually be in use. So don't worry about it for the moment. :+1:

Reply:
Oke, i won't then. Anyway it's really pain in the ass to localize some of these words.